pkg/pillar/types/zedroutertypes.go

```diff
@@ -1924,15 +1967,13 @@ type NetworkInstanceInfo struct {
 	// interface names for the Logicallabel
 	IfNameList []string // Recorded at time of activate
 
-	// Collection of address assignments; from MAC address to IP address
-	IPAssignments map[string]net.IP
-
 	// Union of all ipsets fed to dnsmasq for the linux bridge
 	BridgeIPSets []string
 
 	// Set of vifs on this bridge
 	Vifs []VifNameMac
 
+	IPAssignments map[string]net.IP
```

Comment:
Why did you move this down?
Can you review your diffs against master to make sure that isn't spurious/unneeded changed like this move.